### PR TITLE
gpl: remove unecessary log messages

### PR DESCRIPTION
**Pull request by gudeh**
This PR should resolve issue https://github.com/The-OpenROAD-Project/OpenROAD/issues/6324

### CODE REVIEW

File: src/gpl/src/nesterovBase.cpp [hunk 1]
```diff
@@ -1438,18 +1437,12 @@ void NesterovBaseCommon::fixPointers()
         if (it != db_iterm_map_.end()) {
           size_t gpin_index = it->second;
           gCell.addGPin(&gPinStor_[gpin_index]);
-        } else {
-          log_->report("error: gpin nullptr (from iterm:{}) in gcell:{}",
-                       iterm->getName(),
-                       gCell.instance()->dbInst()->getName());
```
Comment (eder-matheus) [inline src/gpl/src/nesterovBase.cpp:1444]: Do you think it could be useful as a debug print?

**Comment by gadfort** [comment]
@gudeh there are also some warnings a little lower in the file, do you intend to remove those too?

**Comment by gudeh** [comment]
@gadfort have you noticed them being reported? 

I am sorry, I left them unattended because I did not notice any of these being reported, plus I did not want to remove useful checks for potential issues. I believe leaving them as a debugprint is the best approach here.

**Comment by gadfort** [comment]
@gudeh I have not seen them, but to be consistent they should be removed or made debugPrint (again nothing a user can do with those messages)

**Review by eder-matheus — APPROVED** [review]
Looks good to me. @gadfort do you have other comments?

**Comment by gadfort** [comment]
@eder-matheus looks good to me as well

**Pull request merged**